packages/compass-query-bar/src/components/query-bar.tsx

```diff
@@ -44,11 +44,20 @@ const queryBarFormDarkStyles = css({
 
 const queryBarFirstRowStyles = css({
   display: 'flex',
-  alignItems: 'center',
+  alignItems: 'flex-start',
```

Comment:
For context I think when we were looking at this with Maurizio one of the reason we replaced was because it's already a default value for flex parent and there was no context why do we have it, so would be nice to add a comment here explaining why we are setting default flex alignment explicitly so that we don't remove it again

Reply:
Good call! Added.